### PR TITLE
Release 0.0.1

### PR DESCRIPTION
**Pull request by ml-holdenporter**

### CODE REVIEW

**Review by ChristopherWBoyd — DISMISSED** [review]

File: cloudfront.tf [hunk 1]
```diff
@@ -0,0 +1,79 @@
+locals {
+  s3_origin_id = "${var.project_name}-primarys3Origin"
```
Comment (80bower) [inline cloudfront.tf]: > primarys3Origin

Change to `primary-s3-origin`?

File: cloudfront.tf [hunk 2]
```diff
@@ -0,0 +1,79 @@
...
+resource "aws_cloudfront_distribution" "s3_distribution" {
+  aliases = ["${var.domain_alias}"]
+
+  origin {
+    domain_name = aws_s3_bucket.east_website_bucket.bucket_regional_domain_name
+    origin_id   = local.s3_origin_id
+
+    origin_shield {
+      enabled              = true
+      origin_shield_region = "us-east-1"
```
Comment (80bower) [inline cloudfront.tf]: Variable?

File: s3.tf [hunk 3]
```diff
@@ -0,0 +1,36 @@
+resource "aws_s3_bucket" "east_website_bucket" {
+  bucket        = "e1-${var.project_name}"
```
Comment (80bower) [inline s3.tf]: Maybe adjust this prefix.

File: s3.tf [hunk 4]
```diff
@@ -0,0 +1,36 @@
...
+  }
+}
+
+resource "aws_s3_bucket_policy" "east_bucket_read_policy" {
+  bucket = aws_s3_bucket.east_website_bucket.id
+  policy = data.aws_iam_policy_document.east_bucket_policy.json
+}
+
+resource "aws_s3_bucket" "cloudfront_logging" {
+  bucket        = "e1-cloudfront-${var.project_name}-logging"
```
Comment (80bower) [inline s3.tf]: Adjust prefix

**Review by 80bower — CHANGES_REQUESTED** [review]

File: s3.tf (hunk 4 above)
Comment (ml-holdenporter) [reply s3.tf]: Planning on adjusting these, just testing this branch for now.

**Review by ChristopherWBoyd — DISMISSED** [review]

**Review by 80bower — DISMISSED** [review]

**Review by 80bower — APPROVED** [review]

**Pull request merged**